packages/core/src/driver.ts

```diff
@@ -138,6 +138,9 @@ class SessionConfig {
      * straightforward way and potentially simplifies driver logic as well as
      * reduces network communication resulting in better performance.
      *
+     * Usage of Cypher clauses like USE is not a replacement for this option.
+     * Driver sends Cypher to the server for processing.
```

Comment:
```suggestion
     * The driver does not parse any Cypher.
```

maybe?
(not repeating the suggestion for the deno code)